Compiler Error CS0136

Hello,

After importing the package I got these two errors:

> Library/PackageCache/com.bennykok.unity-searchable-event-drawer@d1327b7a23/Editor/GenericAdvancedDropdown.cs(153,73): error CS0136: A local or parameter named 'item' cannot be declared in this scope because that name is used in an enclosing local scope to define a local or parameter


>Library/PackageCache/com.bennykok.unity-searchable-event-drawer@d1327b7a23/Editor/AdvanceDropdownEventDrawer.cs(527,78): error CS0136: A local or parameter named 'c' cannot be declared in this scope because that name is used in an enclosing local scope to define a local or parameter

--- a/Editor/AdvanceDropdownEventDrawer.cs
+++ b/Editor/AdvanceDropdownEventDrawer.cs
@@ -524,7 +524,7 @@ namespace BennyKok.EventDrawer.Editor
 
                             var type = content.text.Substring(0, content.text.IndexOf("/"));
 
-                            if (!stringToTypeCache.TryGetValue(type, out var c))
+                            if (!stringToTypeCache.TryGetValue(type, out var value))
                             {
                                 Type t = null;
                                 var types = TypeCache.GetTypesDerivedFrom<Object>();
@@ -536,14 +536,14 @@ namespace BennyKok.EventDrawer.Editor
                                         break;
                                     }
                                 }
-                                c = EditorGUIUtility.ObjectContent(null, t);
-                                if (c.image == null)
-                                    c = EditorGUIUtility.IconContent("cs Script Icon");
+                                value = EditorGUIUtility.ObjectContent(null, t);
+                                if (value.image == null)
+                                    value = EditorGUIUtility.IconContent("cs Script Icon");
 
-                                stringToTypeCache.Add(type, c);
+                                stringToTypeCache.Add(type, value);
                             }
 
-                            return c.image as Texture2D;
+                            return value.image as Texture2D;
 
                         }).Dropdown(functionRect, 10);
                     }

--- a/Editor/GenericAdvancedDropdown.cs
+++ b/Editor/GenericAdvancedDropdown.cs
@@ -150,7 +150,7 @@ namespace BennyKok.EventDrawer.Editor
 
                 foreach (var str in splitStrings)
                 {
-                    var foundChildItem = parent.children.FirstOrDefault(item => item.name == str);
+                    var foundChildItem = parent.children.FirstOrDefault(children => children.name == str);
 
                     if (foundChildItem != null)
                     {

